### PR TITLE
auth: add root user and root role

### PR DESCRIPTION
**Pull request by xiang90**
automatically create root user with root role when enabling auth if root user does not exist.

### CODE REVIEW

**Comment by gyuho** [comment]
lgtm. /cc @mitake

**Comment by mitake** [comment]
@xiang90 how about simply return an error to the enabling request in a case of no root account like v2 auth? Creating with root account with fixed password seems to be a little bit dangerous, I think.

**Comment by xiang90** [comment]
@mitake We can do that too. I will change.

File: auth/store.go [hunk 1]
```diff
@@ -47,6 +47,12 @@ var (
 	ErrPermissionNotGranted = errors.New("auth: permission is not granted to the role")
 )
 
+const (
+	rootUser      = "root"
+	rootDefaultPW = "root"
+	rootRole      = "rootRole"
```
Comment (mitake) [inline auth/store.go]: Is rootRole needed? I think just creating root account is enough.

Comment (xiang90) [inline auth/store.go]: we have root role that can assign to other users in v2.

Comment (mitake) [inline auth/store.go]: I see, then it is ok

**Comment by mitake** [comment]
@xiang90 lgtm, thanks. Let's update about root account creation later.

**Comment by xiang90** [comment]
@mitake I actually already changed the workflow. PTAL

**Comment by mitake** [comment]
@xiang90 the change lets some e2e tests fail, but the content looks good to me

**Comment by xiang90** [comment]
@mitake I will fix e2e test.

**Pull request merged**